Dockerfile

```diff
@@ -13,7 +13,10 @@ ARG target_arch
 ENV PATH=/opt/cegcc-${target_arch}/bin:$PATH
 
 RUN apt update && \
-    DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends bison build-essential ca-certificates flex git libgmp-dev libmpc-dev libmpfr-dev texinfo
+    apt upgrade -y && \
+    DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends bison build-essential ca-certificates flex git libgmp-dev libmpc-dev libmpfr-dev texinfo && \
+    apt clean && \
```

Comment:
```suggestion
```
The debian docker image [automatically runs `apt clean`](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105) after every install invocation.